frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/PreviewPane/PreviewPane.unit.spec.tsx

```diff
@@ -5,18 +5,14 @@ import { PreviewPane } from "./PreviewPane";
 const MOCK_PREVIEW_URL = "https://www.example.com/";
 
 type MockPreviewPaneProps = {
-  isTransparent?: boolean;
   hidden?: boolean;
 };
 
-const setup = ({
-  isTransparent = false,
-  hidden = false,
-}: MockPreviewPaneProps = {}) => {
+const setup = ({ hidden = false }: MockPreviewPaneProps = {}) => {
   render(
     <PreviewPane
       previewUrl={MOCK_PREVIEW_URL}
-      isTransparent={isTransparent}
+      backgroundType="no-background"
```

Comment:
maybe this isn't the best name. `no-background` means we're not setting a checkerboard background that is for `background: false`. That means the actual `background` setting is `true` for this case.